src/sas/qtgui/Utilities/DocViewWidget.py

```diff
@@ -207,8 +209,24 @@ def loadHtml(self):
         settings.setAttribute(QtWebEngineCore.QWebEngineSettings.LocalContentCanAccessFileUrls, True)
         self.webEngineViewer.load(url)
 
+        # Set widget title to include the name of the document loaded
+        self.updateTitle()
+
```

Comment:
This might cause a 2nd running of `updateTitle()` . Line 210 should trigger the call as well if the urlChanged connection is working properly.